Automattic/wp-calypso
# Remove pricing v1 and v2

### Changes proposed in this Pull Request

This PR removes the deprecated v1 and v2 versions of the pricing page. Current variations include i5 and SPP.

### Implementation notes

Here's a more readable recap of the changes:

- Removed iterations in `client/my-sites/plans/jetpack-plans/abtest.ts`
- Removed following components:
    - ` client/components/jetpack/card/jetpack-free-card-alt`
    - `client/components/jetpack/card/jetpack-product-card-alt`
    - `client/components/jetpack/card/jetpack-product-card-alt-2`
    - `client/my-sites/plans/jetpack-plans/plans-filter-bar`
    - `client/my-sites/plans/jetpack-plans/product-card-alt`
    - `client/my-sites/plans/jetpack-plans/product-card-alt-2`
    - `client/my-sites/plans/jetpack-plans/products-grid-alt`
    - `client/my-sites/plans/jetpack-plans/products-grid-alt-2`
    - `client/my-sites/plans/jetpack-plans/records-details-alt`
- Removed copy and bits of code that were only run with v1 and v2

### Testing instructions

- Download the PR
- Run both Calypso and cloud concurrently
- Select the `control` variation of the `jetpackSimplifyPricingPage` experiment
- In both platforms, check that the plans/pricing page looks and behaves as it does in production
- Select the `test` variation
- Repeat
